### ISSUE TITLE
PhysicalBoneEditor is not properly freed

### ISSUE THREAD

**Issue by akien-mga** [godotengine/godot#22086]
**Godot version:**
Master branch, 9b97c74f

**OS/device including version:**
Mageia 6 x86_64.

**Issue description:**
When closing the editor, the following warning is printed (in verbose mode):
```
WARNING: cleanup: ObjectDB Instances still exist!
   At: core/object.cpp:2081.
Leaked instance: PhysicalBoneEditor:15428
Orphan StringName: PhysicalBoneEditor
StringName: 1 unclaimed string names at exit
```

Might be related to https://github.com/godotengine/godot/blob/9eb4d4ab2d65c7e29b07308c7f18f40b6530f5a2/editor/plugins/physical_bone_plugin.cpp#L73

CC @AndreaCatania 

**Steps to reproduce:**
- Open any project in the editor, in verbose mode
- Close it

**Minimal reproduction project:**
Not necessary.

**Comment by vnen** [comment]
Same as #19482, no?

**Comment by akien-mga** [comment]
Yes, but my report is better :P

**Comment by AndreaCatania** [comment]
I can't reproduce it, with -dev

I get only this one that is not related to this problem I think:

```
WARNING: cleanup: ObjectDB Instances still exist!
   At: core/object.cpp:2081.
ERROR: clear: Resources Still in use at Exit!
   At: core/resource.cpp:425.
ERROR: cleanup: There are still MemoryPool allocs in use at exit!
   At: core/dvector.cpp:70.
```


#855a440071897f757bcd60cf39008b1e374eab06
I've removed the node but I need that you confirm that this fix the issue

**Comment by akien-mga** [comment]
You should run godot with the `-v` / `--verbose` flag. It's still reproducible for me on any project:
```
godot-git -e --path /home/akien/tmp/godot/test/v3.1_test --verbose
XInput: No touch devices found.
Using GLES3 video driver
OpenGL ES 3.0 Renderer: GeForce GTX 670MX/PCIe/SSE2
PulseAudio: detected 2 channels
PulseAudio: audio buffer frames: 512 calculated latency: 11ms
Boot splash path: res://black2x2.png
WARNING: load: Loaded resource as image file, this will not work on export: 'res://black2x2.png'. Instead, import the image file as an Image resource and load it normally as a resource.
   At: core/image.cpp:1734.
Loading resource: res://default_bus_layout.tres
CORE API HASH: -7101670353634600885
EDITOR API HASH: -2162447551495730378
Loading resource: /home/akien/.config/godot/editor_settings-3.tres
EditorSettings: Load OK!
Loaded builtin certs
EditorSettings: Save OK!
Loading resource: res://Node2D.tscn
Loading resource: res://Tree.gd
Loading resource: res://Node2D.gd
[...]
Loading resource: res://Node.vs (cached)
Loading resource: res://Node2D.vs
EditorSettings: Save OK!
EditorSettings: Save OK!
WARNING: cleanup: ObjectDB Instances still exist!
   At: core/object.cpp:2081.
Leaked instance: PhysicalBoneEditor:15458
Orphan StringName: PhysicalBoneEditor
StringName: 1 unclaimed string names at exit.
```

**Comment by akien-mga** [comment]
It tried 855a440 and it doesn't fix it. I'm not sure it's needed to `remove_child` in the first place before `memdelete`-ing, maybe this TODO can be removed altogether.

**Comment by AndreaCatania** [comment]
Thanks you I fixed it: #22339
